templates/test/infopanel.html.ep

```diff
@@ -95,7 +95,7 @@
                         % if ($job->state eq 'running') {
                             <div id="developer-instructions" class="developer-mode-element" data-visible-on="isPaused">
                                 <span title="Developer mode available">
-                                    System is waiting for developer, connect to <b><%= $worker->host %> at port <%= 90 + $worker->instance %> with Shared mode (otherwise the job might be cancelled)</b>
+                                    System is waiting for developer, connect through VNC to <b><%= $worker->host %> at port <%= 5090 + $worker->instance %> with Shared mode (otherwise the job might be cancelled)</b>
```

Comment:
There's still the brace with 'otherwise the job might be canceled'. According to the feedback it wasn't clear what it means. And wouldn't the job also timeout if you connect?

Reply:
Good point. Shall I make it a popup window?

Reply:
Either just remove the brace or explain it more detailed. For the 2nd option a help popover would make sense.

Reply:
just remove it - the job isn't cancelled, the vncviewer will just refuse to connect

Reply:
Ok